<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaFinalNivel</class>
 <widget class="QWidget" name="PantallaFinalNivel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="fondo" native="true">
     <property name="styleSheet">
      <string notr="true">#fondo {
	border-image: url(:/Resources/escritorio2.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QWidget" name="titulos" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="title">
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dia Terminado</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantPersAcept_title">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cantidad personas aceptadas:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantPersAcept_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantPersRech_title">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cantidad personas rechazadas:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantPersRech_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantMultasObt_title">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cantidad multas obtenidas:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cantMultasObt_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="credsSocsObtDia_title">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Creditos Sociales obtenidos en el dia:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="credsSocsObtDia_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="credsSocsTot_title">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Creditos sociales totales:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="credsSocsTot_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="botones" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>100</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="salirBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sigNivelBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>MS Sans Serif</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);</string>
           </property>
           <property name="text">
            <string>Siguiente Nivel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
